freezegun/api.py

```diff
@@ -598,7 +598,10 @@ def tearDownClass(cls):
                         continue
                     seen.add(attr)
 
-                    if not callable(attr_value) or inspect.isclass(attr_value):
+                    # staticmethods are callable from Python 3.10 . Hence skip them from decoration
+                    if (not callable(attr_value)
+                        or inspect.isclass(attr_value)
+                        or isinstance(attr_value, staticmethod)):
```

Comment:
```suggestion
                            or inspect.isclass(attr_value)
                            or isinstance(attr_value, staticmethod)):
```

I don't think you're really supposed to use the same indent level as the console inside `if`.